apps/web/src/views/BuyCrypto/components/AccordianDropdown/AccordianItem.tsx

```diff
@@ -184,7 +187,7 @@ function AccordionItem({
           </Text>
         </RowBetween>
 
-        <DropdownWrapper ref={contentRef}>
+        <DropdownWrapper ref={contentRef} isClicked={!isActive()}>
```

Comment:
Same here, we can use precomputed value instead of making unnecessary function call